OpenRA.Mods.Cnc/Traits/PoisonedByTiberium.cs

```diff
@@ -49,7 +49,7 @@ public void Tick(Actor self)
 			if (r == null || !Info.Resources.Contains(r.Info.Name))
 				return;
 
-			Info.WeaponInfo.Impact(Target.FromActor(self), self.World.WorldActor, Enumerable.Empty<int>());
+			Info.WeaponInfo.Impact(Target.FromPos(self.CenterPosition), self.World.WorldActor, Enumerable.Empty<int>());
```

Comment:
It crashes with the exceptions I mentioned in https://github.com/OpenRA/OpenRA/pull/10813/commits/7b80114e9c6e3cfcbe71a3c4dd1df5026e40f975.